dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05350AddDotSaltClusterColumnTest.java

```diff
@@ -0,0 +1,52 @@
+package com.dotmarketing.startup.runonce;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Logger;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05350AddDotSaltClusterColumnTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    private void dropColumn(final DotConnect dotConnect) throws SQLException {
+        try {
+            final String dropColumnSQL = "ALTER TABLE dot_cluster DROP COLUMN cluster_salt";
+            dotConnect.executeStatement(dropColumnSQL);
+        } catch (Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)